contracts/governancestaking/GovarnanceStaking.sol

```diff
@@ -56,7 +56,7 @@ contract GovernanceStaking is GovernanceStakingToken,Initializable{
 	) public virtual initializer {
 		setDAO(_ns);
 		token = ERC20(address(avatar.nativeToken()));
-        rewardsPerBlock = 7; // 7 Govarnance token per block as reward to distribute 12M token monthly
+        rewardsPerBlock = 7 * 1e18; // 7 Govarnance token per block as reward to distribute 12M token monthly
```

Comment:
let have a function setMonthlyRewards(X) and it will calculate how much is rewardsPerBlock from X

Reply:
setRewardsPerBlock should stay as well?

Reply:
It was 70 I put wrong

Reply:
we can remove setRewardsPerBlock
make the monthly blocks amount in a variable, initialized with input from initialize()